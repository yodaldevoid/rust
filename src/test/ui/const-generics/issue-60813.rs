// run-pass

#![feature(const_generics)]
//~^ WARN the feature `const_generics` is incomplete and may cause the compiler to crash

pub fn function_with_str<const STRING: &'static str>() {}

pub fn main() {
    function_with_str::<"Hello, world!">()
}
