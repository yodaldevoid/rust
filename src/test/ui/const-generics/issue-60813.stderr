warning: the feature `const_generics` is incomplete and may cause the compiler to crash
  --> $DIR/issue-60813.rs:3:12
   |
LL | #![feature(const_generics)]
   |            ^^^^^^^^^^^^^^

